public/app/plugins/datasource/loki/datasource.test.ts

```diff
@@ -271,6 +271,28 @@ describe('LokiDatasource', () => {
         'rate({bar="baz", job="foo", k1=~"v.*", k2=~"v\\\\\'.*"} |= "bar" [5m])'
       );
     });
+
+    it('should interpolate before adding adhoc filters', async () => {
```

Comment:
The test doesn't really show those query with by at the end will work now no ?

Reply:
It was enough for me to know that `templateSrv` was called with the original query, but I refactored the test a bit to make what it's asserting more obvious 👍